springwolf-examples/springwolf-amqp-example/src/main/java/io/github/springwolf/examples/amqp/dtos/AnotherPayloadDto.java

```diff
@@ -19,5 +19,5 @@ public class AnotherPayloadDto {
     private String foo;
 
     @Schema(description = "Example field", requiredMode = REQUIRED)
-    private ExamplePayloadDto example;
+    private GenericPayloadDto<ExamplePayloadDto> example;
```

Comment:
Testing generic payload: generated yaml seems to be good: I was not able to reproduce the "?" issue.
Maybe it's because my 'old SpringBoot 2.x' app comes with old dependencies, which are responsible for the problem.
And in SpringBoot 3.x applications, new version of these dependencies are used -> no more bug.